pythonhelp.py

```diff
@@ -0,0 +1,50 @@
+
+class FizzBuzz(object):
+    def __init__(self):
```

Comment:
Could we pass the final number into the object when we initialise it? That way the methods below won't need to be passed a `number` - they can just access it as an internal (if that makes sense).